bot/seasons/evergreen/issues.py

```diff
@@ -3,55 +3,60 @@
 import discord
 from discord.ext import commands
 
-from bot.constants import Colours
-from bot.decorators import override_in_channel
-
 log = logging.getLogger(__name__)
 
+icons = {"issue": "https://i.imgur.com/HFV0nv9.png",
+         "issue-closed": "https://i.imgur.com/uZSX7as.png",
+         "pull-request": "https://i.imgur.com/zHhnALX.png",
+         "pull-request-closed": "https://i.imgur.com/JYmTn5P.png",
+         "merge": "https://i.imgur.com/xzQZxXe.png"}
+
+respValue = {404: "Issue/pull request not located! Please enter a valid number!",
```

Comment:
For constants, PEP8 [specifies](https://www.python.org/dev/peps/pep-0008/#constants) all-capitals with underscores separating words.